v1/pdf.yaml

```diff
@@ -53,27 +53,27 @@ paths:
           description: Error
           schema:
             $ref: '#/definitions/problem'
-      operationId: 'generatePDF'
-#      x-request-handler:
-#        - get_latest_revision:
-#            request:
-#              method: get
-#              uri: /{domain}/sys/page_revisions/page/{title}
-#        - get_pdf_from_backend:
-#            request:
-#              method: get
-#              # Note: The title needs to be encoded twice.
-#              uri: '{{options.uri}}/pdf?accessKey={options.secret}&url={{default(options.scheme, "https")}}://{{domain}}/wiki/{_encodeURIComponent(title)}%3Fprintable=yes'
-#            return:
-#              status: 200
-#              headers:
-#                # Firefox supports filename*= syntax while Chrome respect
-#                # filename=. Safari is stupid and understands neither.
-#                # TODO: Quote raw `"` chars in filename.
-#                content-disposition: 'attachment; {{filenameParameters(request.params.title)}}'
-#                content-type: '{{get_pdf_from_backend.headers.content-type}}'
-#                content-length: '{{get_pdf_from_backend.headers.content-length}}'
-#                cache-control: '{{default(options.cache_control, "s-maxage=600, max-age=600")}}'
-#                etag: '{{get_latest_revision.headers.etag}}'
-#              body: '{{get_pdf_from_backend.body}}'
+      x-request-handler:
+        - get_latest_revision:
+            request:
+              method: get
+              uri: /{domain}/sys/page_revisions/page/{title}
+        - get_pdf_from_backend:
+            request:
+              method: get
+              # Note: The title needs to be encoded twice.
+              # uri: '{{options.uri}}/pdf?accessKey={options.secret}&url={{default(options.scheme, "https")}}://{{domain}}/wiki/{_encodeURIComponent(title)}%3Fprintable=yes'
+              uri: '{{options.uri}}/{{default(options.scheme, "https")}}://{{domain}}/v1/pdf/{_encodeURIComponent(title)}/a4/desktop'
+            return:
+              status: 200
+              headers:
+                # Firefox supports filename*= syntax while Chrome respect
+                # filename=. Safari is stupid and understands neither.
+                # TODO: Quote raw `"` chars in filename.
+                content-disposition: 'attachment; {{filenameParameters(request.params.title)}}'
```

Comment:
Sice there's no pdf.js anymore that has been attaching the filenameParameters function to the context of the template expansion this will certainly not work.

Reply:
Readded